### ISSUE TITLE
Unicode characters support

### ISSUE THREAD

**Issue by ferchitas** [tarql/tarql#80]
I am trying to get some triples csv file that has words write in Greek Alphabet but I do not get any triple. However, if I translate that greek letters to Latin Alphabet I get the expected output, that means the sparql query is well defined. Is there any solution to this already implemented or an easy way to solve it?

If you need extra information just ask.

**Comment by cygri** [comment]
Did you try the `-e` argument on the command line to specify Greek character encoding? Probably `-e iso-8859-7` or `-e windows-1253`?

**Comment by ferchitas** [comment]
Yes, I tried both of them, but the result is the same. Manually, I changed the file encoding with notepad++, but, as I said, nothing has changed.

**Comment by cygri** [comment]
Did you try `-e utf-8` as well? Can you reduce the problem to a single-line CSV file that you could share here?
